docs/siren_system/index.md

```diff
@@ -0,0 +1,39 @@
+
+# Siren system
```

Comment:
Unsure if you should list all the index of the subfiles again here.

Reply:
I ended up taking this approach because wanted to make sure people would have an overview of what is advanced usage and what is further modding. 
> I want to change light animations
Go to the index, it's there, under advanced usage 
If I linked only the subfiles, people would have to wonder which file contains what they are looking for.

This makes the index kind of cluttered, though. So either approach is good for me.

Even better would be to make a FAQ

Reply:
Well If I want to change the light animations I will look at the 3 available pages and their short description and see which one appears to be what I need. I think the index.md should contain not more than than. Especially in regards of if something is changed in the sub change (like a heading is added) then you also always need to edit the main index ...